src/pytest_dbfixtures/factories/postgresql.py

```diff
@@ -121,7 +121,7 @@ def drop_postgresql_database(psycopg2, user, host, port, db):
     conn = psycopg2.connect(user=user, host=host, port=port)
     conn.set_isolation_level(psycopg2.extensions.ISOLATION_LEVEL_AUTOCOMMIT)
     cur = conn.cursor()
-    cur.execute('DROP DATABASE IF EXISTS %s' % db)
+    cur.execute('DROP DATABASE IF EXISTS %s;' % db)
```

Comment:
We have `cur.execute('CREATE DATABASE ' + db)`. Also the `DROP DATABASE` works (I do get errors from it if a buggy test leaves an open database connection). I cannot find explicit rules about semicolons, but it seems the single statement passed via `cur.execute` doesn't need a trailing one.